index.js

```diff
@@ -33,8 +38,10 @@ Plugins.prototype = Emitter({
       next = opts;
       opts = null;
     }
-    argv = this.minimist(argv, opts);
-    this.plugins.run(argv, function (err, args) {
```

Comment:
That gives plugins more power and ability control minimist options, which in some cases may be useful. Say "if something, so I dont won't alias of `--set` -> remove it from options", or something like that. That's usefulness, because of that i think for it.